modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FrontendDataSetAdmin.macro

```diff
@@ -258,6 +274,12 @@ definition {
 		Click(locator1 = "FrontendDataSetAdmin#VIEW_ENTRY_BUTTON");
 	}
 
+	macro saveFields {
+		Click(locator1 = "FrontendDataSetAdmin#MODAL_SAVE");
+
+		WaitForLiferayEvent.initializeLiferayEventLog();
+	}
+
```

Comment:
Try to use Button#SAVE instead of creating a new path. To avoid selecting the wrong Save button, try to use SelectFrame to assert the correct button